etl/steps/data/garden/faostat/2024-03-14/faostat_qcl.meta.yml

```diff
@@ -45,3 +48,1345 @@ tables:
+        title: Beans, dry | 00000176 || Yield | 005419 || tonnes per hectare
+        unit: tonnes per hectare
+        short_unit: t/ha
+        processing_level: major
+        display:
+          name: Bean yields
+          shortUnit: t/ha
+          numDecimalPlaces: 2
+        presentation:
+          title_public: Bean yields
+      cashew_nuts__00000217__production__005510__tonnes:
+        title: Cashew nuts | 00000217 || Production | 005510 || tonnes
+        unit: tonnes
+        short_unit: t
+        processing_level: major
+        display:
+          name: Cashew nut production
+          shortUnit: t
+          numDecimalPlaces: 2
+        presentation:
+          title_public: Cashew nut production
+      cashew_nuts__00000217__yield__005419__tonnes_per_hectare:
+        title: Cashew nuts | 00000217 || Yield | 005419 || tonnes per hectare
+        unit: tonnes per hectare
+        short_unit: t/ha
+        processing_level: major
+        display:
+          name: Cashew nut yields
+          shortUnit: t/ha
+          numDecimalPlaces: 2
+        presentation:
+          title_public: Cashew nut yields
+      cassava__00000125__area_harvested__005312__hectares:
+        title: Cassava | 00000125 || Area harvested | 005312 || hectares
+        unit: hectares
+        short_unit: ha
+        processing_level: major
+        display:
+          name: Land used for cassava production
+          shortUnit: ha
+          numDecimalPlaces: 2
+        presentation:
+          title_public: Land used for cassava production
+      cassava__00000125__production__005510__tonnes:
+        title: Cassava | 00000125 || Production | 005510 || tonnes
+        unit: tonnes
+        short_unit: t
+        processing_level: major
+        display:
+          name: Cassava production
+          shortUnit: t
+          numDecimalPlaces: 2
+        presentation:
+          title_public: Cassava production
+      cassava__00000125__yield__005419__tonnes_per_hectare:
+        title: Cassava | 00000125 || Yield | 005419 || tonnes per hectare
+        unit: tonnes per hectare
+        short_unit: t/ha
+        processing_level: major
+        display:
+          name: Cassava yields
+          shortUnit: t/ha
+          numDecimalPlaces: 2
+        presentation:
+          title_public: Cassava yields
+      castor_oil_seed__00000265__area_harvested__005312__hectares:
+        title: Castor oil seed | 00000265 || Area harvested | 005312 || hectares
+        unit: hectares
+        short_unit: ha
+        processing_level: major
+        display:
+          name: Land used for castor oil seed production
+          shortUnit: ha
+          numDecimalPlaces: 2
+        presentation:
+          title_public: Land used for castor oil seed production
+      cattle__00000866__stocks__005111__animals:
+        title: Cattle | 00000866 || Stocks | 005111 || animals
+        unit: animals
+        short_unit: animals
```

Comment:
I wouldn't have a short unit for this, I think 'animals' is too long and doesn't add necessary information.